src/Simulation/qdk_sim_rs/.gitignore

```diff
@@ -4,8 +4,6 @@ win10
 target
 drop
 
-# We inject version numbers into Cargo.toml, so don't want them stored in repo.
-Cargo.toml
```

Comment:
was this removed on purpose?

Reply:
Yes, this is part of a change in the change done by @cgranade to avoid the .NET5 dependency that had been introduced by Cargo Edit.